definitions/checks/repositories/validate.rb

```diff
@@ -18,19 +18,29 @@ class Validate < ForemanMaintain::Check
     end
 
     def run
-      with_spinner("Validating availability of repositories for #{@version}") do |spinner|
-        absent_repos = feature(:downstream).absent_repos(@version)
-        unless absent_repos.empty?
-          spinner.update('Some repositories missing, calling `subscription-manager refresh`')
-          feature(:downstream).rhsm_refresh
-          absent_repos = feature(:downstream).absent_repos(@version)
-        end
-        unless absent_repos.empty?
-          fail!(
-            "Following repositories are not available on your system: #{absent_repos.join(', ')}"
-          )
+      if feature(:instance).subscribed_using_activationkey?
```

Comment:
Why is this check in `:instance` and not in `:downstream` feature?

Reply:
Yes, this could be downstream only. I'm ok with moving that

Reply:
@mbacovsky, updated PR with a suggested change.